src/Core/Parser/Sequencer.php

```diff
@@ -191,13 +196,14 @@ public function sequence(): ComponentInterface
                     try {
                         $childNode = $this->sequenceTagNode();
                     } catch (Exception $exception) {
+                        if (!$exception instanceof SequencingException) {
```

Comment:
This changed block seems redundant - or am I overlooking something? If redundant, please revert this change block.